fxa-auth-db-server/index.js

```diff
@@ -107,6 +107,9 @@ function createServer(db) {
   api.get('/securityEvents/:id/ip/:ipAddr', withParams(db.securityEvents))
   api.post('/securityEvents', withBodyAndQuery(db.createSecurityEvent))
 
+  api.get('/emailBounces/:email', op(req => db.fetchEmailBounces(req.params.email)))
```

Comment:
FWIW, I don't mind picking one and sticking with it. We can run a sed script on the content server if we decide `foo => ...` is an easier way to go.